test/mocks/server/mocks.h

```diff
@@ -77,6 +77,7 @@ class MockOptions : public Options {
   MOCK_CONST_METHOD0(componentLogLevels,
                      const std::vector<std::pair<std::string, spdlog::level::level_enum>>&());
   MOCK_CONST_METHOD0(logFormat, const std::string&());
+  MOCK_CONST_METHOD0(logFormatEscapeNewlines, bool());
```

Comment:
Would it be possible to write some tests based on the `EXPECT_LOG_CONTAINS*` macros? I'm guessing it might be begging the question, since the macro itself uses a dedicated sink delegate, but might be worth noodling on whether this is possible.

Reply:
I looked into this and might actual be able to do this! I'll work on this soon